src/xcp_ng_dev/files/Dockerfile-8.x

```diff
@@ -19,18 +19,15 @@ RUN     sed -i -e "s/@XCP_NG_BRANCH@/${XCP_NG_BRANCH}/g" /etc/yum.repos.d/xcp-ng
 # Install GPG key
 RUN     curl -sSf https://xcp-ng.org/RPM-GPG-KEY-xcpng -o /etc/pki/rpm-gpg/RPM-GPG-KEY-xcpng
 
-# Fix invalid rpmdb checksum error with overlayfs, see https://github.com/docker/docker/issues/10180
-# (still needed?)
-RUN     yum install -y yum-plugin-ovl
-
-# Use priorities so that packages from our repositories are preferred over those from CentOS repositories
-RUN     yum install -y yum-plugin-priorities
-
 # Update
-RUN     yum update -y
-
-# Common build requirements
-RUN     yum install -y \
+RUN     yum update -y \
+        # Fix invalid rpmdb checksum error with overlayfs, see https://github.com/docker/docker/issues/10180
+        # (still needed?)
+        && yum install -y yum-plugin-ovl \
+        # Use priorities so that packages from our repositories are preferred over those from CentOS repositories
+        && yum install -y yum-plugin-priorities \
+        # Common build requirements
+        && yum install -y \
```

Comment:
Rather than this change, which makes the commands uglier, a simple `$docker build --squash` shows the same size reduction

Reply:
`docker/build-push-action` doesn't support the squash feature.
If this is too ugly, we could move it to an external script?